tests/gui/test_dialog_model.py

```diff
@@ -224,17 +222,13 @@ def test_load_configuration_init(mock_rivers_object: MagicMock, mocker) -> None:
     Test case for loading configuration initialization.
 
     given: A DialogModel instance, mock RiversObject, and mocker.
-    when: Initializing the DialogModel with a configuration file.
+    when: Initializing the DialogModel.
     then: The configuration object is not None.
     """
-    filename = "test.cfg"
     config = ConfigParser()
     config["General"] = {}
-    mock_load_config = mocker.patch.object(
-        ConfigFileOperations, "load_configuration_file", return_value=config
-    )
-    model = DialogModel(mock_rivers_object, filename)
-    mock_load_config.assert_called_once_with(filename)
+    model = DialogModel(mock_rivers_object)
+
```

Comment:
The name of this tests should be updated, since you are not loading the config anymore.

Reply:
Test can be removed. @MRVermeulenDeltares create PR.

Reply:
See PR #107

Reply:
Discussed with Bert.